src/rospack.cpp

```diff
@@ -325,45 +325,40 @@ Rosstackage::isStackage(const std::string& path)
   return false;
 }
 
+static bool sameSearchPaths(const std::vector<std::string> &path1,
+                            const std::vector<std::string> &path2)
+{
+  if(path1.size() != path2.size())
+    return false;
+
+  for(unsigned int i=0, end=path1.size(); i != end; ++i)
+  {
+    if(path1[i] != path2[i])
+      return false;
+  }
+  return true;
+}
```

Comment:
How is this function different from the comparison operator of a vector (http://en.cppreference.com/w/cpp/container/vector/operator_cmp)?